components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -667,7 +667,12 @@ const ContributorHighlightCard = ({
                   <Emoji className="text-xs text-orange-500 md:text-sm" text={`:${getEmojiNameById(emoji_id)}:`} />
                   <span>{reaction_count}</span>
                 </span>
-              </button>
+        </button>
+                <Emoji
+                  className="text-xs text-orange-500 md:text-sm"
+                  text={`:${getEmojiNameById(emoji_id)}: ${reaction_count}`}
+                />
+              </div>
```

Comment:
@CBID2 we need to remove this. i think this came from the merge conflict commit.

Reply:
Good idea @Arol15. You have my permission to do this :)

Reply:
hmm, i think the above lines (667-669) are the old code. I'll double-check. don't remove it yet.